Soccar [Game-Server]/src/nl/soccar/gameserver/model/PlayerWrapper.java

```diff
@@ -8,18 +8,31 @@
 import nl.soccar.socnet.connection.Connection;
 
 /**
+ * Wrapper for the player class.
+ * 
  * @author PTS34A
  */
 public final class PlayerWrapper {
 
     private final Connection connection;
     private final Player player;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)